src/platform/endpoint/test/node/openaiCompatibleEndpoint.ts

```diff
@@ -105,26 +107,40 @@ export class OpenAICompatibleTestEndpoint extends ChatEndpoint {
 			throw new Error(`API key environment variable ${this.modelConfig.apiKeyEnvName} is not set`);
 		}
 
+		if (this.modelConfig.type === 'azureOpenai') {
```

Comment:
Should this be in a separate azure endpoint? Just in case they continue to diverge? Maybe via na override. You can see BYOK does this